docs/platform-hyperkit.md

```diff
@@ -91,6 +91,22 @@ docker build -t socat .
 docker run --rm -t -d -p 6379:6379 socat tcp-listen:6379,reuseaddr,fork tcp:<IP address of VM>:6379
 ```
 
+#### Port forwarding with `vpnkit`
+
+VPNKit has the general tooling to expose any guest VM port on the host (just
+like it does with containers in Docker for Mac). To enable forwarding, a
+`vpnkit-forwarder` container must be running in the VM. The VM also has to be
+booted with `linuxkit run hyperkit -networking=vpnkit`. A `vpnkit-expose-port`
+container has to run for each exposed port. The containers currently use a 9P
```

Comment:
is there some way in future we can use a single container to expose multiple ports?